setup.py

```diff
@@ -0,0 +1,38 @@
+# Conversion from Markdown to pypi's restructured text: https://coderwall.com/p/qawuyq -- Thanks James.
+
+try:
+    import pypandoc
+    long_description = pypandoc.convert('README.md', 'rst')
+except (IOError, ImportError):
+    long_description = ''
+    print 'warning: pandoc or pypandoc does not seem to be installed; using empty long_description'
+
+import importlib
+from pip.req import parse_requirements
+from setuptools import setup
+
+install_requires = parse_requirements('requirements.txt', session=False)
+install_requires = [str(ir.req) for ir in install_requires]
+
+setup(
+    name='py_windows_exe',
+    version=importlib.import_module('py_windows_exe').__version__,
+    author='Body Labs',
+    author_email='___@bodylabs.com',
```

Comment:
Why don't you put me as the contact on this one, since I seem to get all the python on windows pain here ;)